PusherMainView/PusherMainView/Base.lproj/Pusher.storyboard

```diff
@@ -191,32 +204,31 @@
                 </viewController>
                 <customObject id="rPt-NT-nkU" userLabel="First Responder" customClass="NSResponder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="75" y="744"/>
+            <point key="canvasLocation" x="75" y="708.5"/>
         </scene>
         <!--Devices View Controller-->
         <scene sceneID="wsB-0v-r0Y">
             <objects>
                 <viewController storyboardIdentifier="DevicesViewController" id="DA4-mJ-h0m" customClass="DevicesViewController" customModule="PusherMainView" customModuleProvider="target" sceneMemberID="viewController">
-                    <view key="view" id="uGH-nk-87p">
-                        <rect key="frame" x="0.0" y="0.0" width="480" height="480"/>
+                    <view key="view" misplaced="YES" id="uGH-nk-87p">
```

Comment:
Misplaced view. Although I think it won't cause any issues in runtime

Reply:
Fixed.